zmq/backend/cffi/_verify.c

```diff
@@ -1,11 +1,21 @@
-#include <stdio.h>
-#include <sys/un.h>
-#include <string.h>
+/*
 
-#include <zmq.h>
-#include "zmq_compat.h"
+Platform-independant detection of IPC path max length
 
+Copyright (c) 2012 Godefroid Chapelle
+
+Distributed under the terms of the New BSD License.  The full license is in
+the file COPYING.BSD, distributed as part of this software.
+ */
+
+#if defined(HAVE_SYS_UN_H)
+#include "sys/un.h"
```

Comment:
it looks like this is now a copy of ipcmaxlen.h. Could we instead load it with `#include "ipcmaxlen.h"`?

Reply:
I have changed a way to fix it as per your advice. Thanks for the idea that I didn't come up with.